appveyor.yml

```diff
@@ -1,7 +1,7 @@
 environment:
 
   TOX_ENV: pywin
-
+  APPVEYOR: true
```

Comment:
hmm, sorry, I was confused - I thought appveyor was skipping some tests and that it was because of the `@only_ci`, because it didn't provide an APPVEYOR env var
but now i tried it, and it does :)
so, removed